src/configurationProvider.ts

```diff
@@ -127,18 +128,18 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
                 }
                 if (this.isEmptyArray(config.classPaths) && this.isEmptyArray(config.modulePaths)) {
                     const hintMessage = "Cannot resolve the modulepaths/classpaths automatically, please specify the value in the launch.json.";
-                    vscode.window.showErrorMessage(hintMessage);
+                    utility.showErrorMessage(hintMessage);
                     this.log("usageError", hintMessage);
                     return undefined;
                 }
             } else if (config.request === "attach") {
                 if (!config.hostName || !config.port) {
-                    vscode.window.showErrorMessage("Please specify the host name and the port of the remote debuggee in the launch.json.");
+                    utility.showErrorMessage("Please specify the host name and the port of the remote debuggee in the launch.json.");
```

Comment:
shall we have the anchor in wiki page and references the right part of the document?

Reply:
This priority is not high, will leave it to another PR.